app/static/tests/unit/components/graphConfig/graphConfig.test.ts

```diff
@@ -93,14 +93,34 @@ describe("GraphConfig", () => {
         const wrapper = getWrapper();
         const s = wrapper.findAll(".graph-config-panel .badge").at(0)!;
         const setData = jest.fn();
-        await s.trigger("dragstart", { dataTransfer: { setData } });
+        await s.trigger("dragstart", { dataTransfer: { setData }, ctrlKey: false, metaKey: false });
         expect(setData.mock.calls[0][0]).toBe("variable");
         expect(setData.mock.calls[0][1]).toStrictEqual("S");
         expect(setData.mock.calls[1][0]).toStrictEqual("srcGraphConfig");
         expect(setData.mock.calls[1][1]).toStrictEqual("0");
+        expect(setData.mock.calls[2][0]).toBe("copyVar");
+        expect(setData.mock.calls[2][1]).toBe("false");
```

Comment:
```suggestion
        expect(setData.mock.calls[0].slice(0, 2)).toBe(["variable", "S"]);
        expect(setData.mock.calls[1].slice(0, 2)).toBe(["srcGraphConfig", "0"]);
        expect(setData.mock.calls[2].slice(0, 2)).toBe(["copyVar", "false"]);
```

Easier to read, if it works? Now I can see that copyVar is meant to relate to "false".

Reply:
That doesn't quite work, but `toHaveBeenNthCalledWith` gives us much the same thing, so switching to that.